docs/Home.md

```diff
@@ -1,3 +1,13 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+The purpose of our android-templates repository is to quickly start a new project with all the **must-have** requirements. Other **nice-to-have** features need to be considered as an optional setting when starting a new project.
+
+Planning to add a new feature? Ask yourself the following questions to define if it's a **must-have** requirement:
+
+- Is the feature frequently used in other projects? (Ex: Glide, Retrofit, ...)
+- Does the feature increase productivity of the team? (Ex: Firebase, Proxyman, ...)
+
+Do _both_ apply to your feature? If not, we will define it as a **nice-to-have** feature:
+
+- Ex: [Jetpack Compose](https://developer.android.com/jetpack/compose) simplifies writing code for UI, but is not frequently used (yet)
+- Ex: [Lottie](https://lottiefiles.com/) is frequently used, but does not provide any other value unless JSON animations are needed
```

Comment:
I think we can interpret it a bit more direct and less political here (it is this, but not...unless...)

How about

```suggestion
- Ex: [Lottie](https://lottiefiles.com/) a library for animation exclusively with a specific json format file. It has a specific use case and requirements that not every project is having.
```